app/assets/javascripts/shared/SharedService.js.coffee

```diff
@@ -27,7 +27,7 @@ SharedService = ($http, $state, $window, $document) ->
   Service.getLanguageCode = (langName) ->
     # will take "English" and return "en", for example
     if langName
-      _.invert(Service.languageMap)[_.startCase(langName)]
+      _.invert(Service.languageMap)[_.startCase(_.capitalize(langName))]
```

Comment:
```suggestion
      _.invert(Service.languageMap)[_.capitalize(langName)]
```

Should just need capitalize and not start case in this case, but we'll see how it does on the tests